en/features_mc/throw_launch.md

```diff
@@ -0,0 +1,92 @@
+# Throw Launch (Multicopter)
+
+:::warning Experimental
+This feature was introduced after PX4 v1.14.
+
+- It has not yet been broadly tested on different vehicle configurations or scenarios. various platforms and in different scenarios.
+- The majority of testing has been done in position mode.
+  Other modes should also work.
+
+:::
+
+This feature allows a multicopter to be started by arming it from a fixed position and then throwing it into the air.
+The vehicle then turns on the motors and operates according to its current mode.
+
+The vehicle detects that it has been thrown based on reaching a certain speed (5m/s), and then starts the motors at the apex of the throw, once it determines that it has started to fall.
```

Comment:
```suggestion
The vehicle detects that it has been thrown based on reaching a certain speed (default 5m/s), and then starts the motors at the apex of the throw, once it determines that it has started to fall.
```

Reply:
I put default because that speed is a default. I guess it doesn't matter since the fact it can be changed is clear lower down.